docs/environment.yml

```diff
@@ -27,6 +27,7 @@ dependencies:
   # list of federated extensions for the demo site
   - ipywidgets
   - bqplot
+  - ipycanvas
```

Comment:
Needs a quick rebase since #156 also modified this line.

Reply:
Fixed!